pydicom/charset.py

```diff
@@ -355,20 +536,35 @@ def _encode_string_parts(value, encodings):
                     max_index = e.start
                     best_encoding = encoding
         # none of the given encodings can encode the first character - give up
-        if best_encoding is None:
+        if max_index == 0:
             raise ValueError()
 
```

Comment:
Would be better to have some kind of message with the `raise`.

Reply:
This is used internally only and handled by the caller, so no need for a message here.

Reply:
I changed to pass the comment on line 538 as an argument of ValueError. What do you think?  I agree to what @mrbean-bremen said. Though, if the leaking of this exception occurs, I think this message will be useful for debugging.

https://github.com/ar90n/pydicom/blob/3c75b67becd7861f959505753394276b8533491c/pydicom/charset.py#L541